Formula/systemd.rb

```diff
@@ -34,35 +34,25 @@ class Systemd < Formula
   depends_on "xz"
   depends_on "zstd"
 
-  # Fix missing mount constants, remove in next version
-  patch do
-    url "https://github.com/systemd/systemd/commit/0764e3a327573e7bda2f0e1a914f28482ab00574.patch?full_index=1"
-    sha256 "ad34441deb22b37272d4fd6a307a804f8ceffc0452c17f2353a144b3c04d5451"
-  end
-
   def install
     ENV["PYTHONPATH"] = Formula["jinja2-cli"].opt_libexec/Language::Python.site_packages("python3")
 
-    args = %W[
-      --prefix=#{prefix}
-      --libdir=lib
+    args = *std_meson_args + %W[
       --sysconfdir=#{etc}
       --localstatedir=#{var}
       -Drootprefix=#{prefix}
-      -Dsysvinit-path=#{prefix}/etc/init.d
-      -Dsysvrcnd-path=#{prefix}/etc/rc.d
-      -Dpamconfdir=#{prefix}/etc/pam.d
+      -Dsysvinit-path=#{etc}/init.d
+      -Dsysvrcnd-path=#{etc}/rc.d
+      -Dpamconfdir=#{etc}/pam.d
```

Comment:
I think @iMichka put these in `prefix/"etc"` on purpose, though I don't know why. They look like configuration directories, though, which definitely belong in `etc`.

Reply:
There are some explanations here https://github.com/Homebrew/linuxbrew-core/issues/5408 and https://github.com/Homebrew/linuxbrew-core/pull/5446, these paths were added to fix a compilation issue found in 2018.

Reply:
It looks like there it just extended the existing `etc` references.

Reply:
Well, it seems to compile fine now. And the dependent failure is definitely unrelated. (Still worth investigating tho)

Reply:
I will submit a fix for the dependent failure in `pyside@2`, which is indeed completely unrelated.